internal/pkg/action/operator_uninstall.go

```diff
@@ -165,6 +167,13 @@ func (u *OperatorUninstall) getInstallPlanResources(ctx context.Context, install
 		if step.Resource.Kind == "ServiceAccount" && obj.GetNamespace() == "" {
 			obj.SetNamespace(installPlanKey.Namespace)
 		}
```

Comment:
This section can be removed since it will be covered by the added `bundle.IsSupported()` block.